assets/js/translations-page.js

```diff
@@ -1,4 +1,7 @@
 jQuery(function($) {
+	var $bulkActions = $( '#bulk-action' ),
+		$bulkPriority = $( '#bulk-priority' );
```

Comment:
No, because of the comma this is the same as `var $bulkActions = …; var $bulkPriority = …;`